webpack/components/ResourceQuotaEmptyState/index.js

```diff
@@ -28,16 +29,32 @@ const ResourceQuotaEmptyState = () => {
       {__('Create resource quota')}
     </Button>
   );
+
+  const description = (
+    <span>
+      {__(
+        'Resource Quotas help admins to manage hardware resources including CPUs, memory, and disk space among users or usergroups.'
```

Comment:
```suggestion
        'Resource Quotas help admins to manage resources including CPUs, memory, and disk space among users or user groups.'
```

suggestion (non-blocking): I'd drop this because it reminds me of bare metal.

In the docs, it's called host resources: https://docs.theforeman.org/nightly/Administering_Project/index-katello.html#limiting-host-resources

Also, user group -> two words.

Reply:
Done, thanks!